kgforge/specializations/stores/bluebrain_nexus.py

```diff
@@ -169,7 +169,16 @@ def retrieve(self, id: str, version: Optional[Union[int, str]] = None) -> Resour
             self.service.sync_metadata(resource, response)
             return resource
 
-    def _download_one(self, url: str, path: Path) -> None:
+    def _retrieve_filename(self, id: str) -> str:
```

Comment:
The filename corresponds to the distribution.name value. Can't that be used instead of getting the file metadata from Nexus ?

Reply:
There is no guarantee that this is the case. Indeed, someone could decide to model their data differently.

This implementation offers:
1. independence with how people model their data,
2. consistency across different Store implementations.

Reply:
Okay. Let proceed with your version.